tests/test_training.py

```diff
@@ -84,8 +96,9 @@ def test_cpu_cli_training(self, load_type):
         self.call_cli_with_overrides("cpu", "32", load_type)
 
     @pytest.mark.ipu
-    @pytest.mark.skip
-    @pytest.mark.parametrize("load_type", ["RAM", "disk"])
+    #@pytest.mark.skip
+    #@pytest.mark.parametrize("load_type", ["RAM", "disk"])
+    @pytest.mark.parametrize("load_type", ["disk"])
```

Comment:
```suggestion
    @pytest.mark.parametrize("load_type", ["RAM", "disk"])
```
Let's do both RAM and disk, and tidy up the comments